contracts/governance/Staking/Staking.sol

```diff
@@ -298,21 +298,89 @@ contract Staking is
     }
 
     /**
-     * @notice Withdraw tokens for vesting contract.
-     * @param vesting The address of Vesting contract.
-     * @param receiver The receiver of the tokens. If not specified, send to the msg.sender
-     * @dev Can be invoked only by whitelisted contract passed to governanceWithdrawVesting.
+     * @notice New governance withdraw vesting directly through staking contract.
```

Comment:
this comment is misleading  
```suggestion
* @notice Governance withdraw vesting through staking contract.
     * This direct withdraw vesting, solves the out of gas issue when there are too many iterations when withdrawing.
     * This function only allows cancelling vesting contract of the TeamVesting type.
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/7f1c269ee40b1d2f3f864b07c574360839a9d270)

Reply:
could you pls also apply the text fix suggestions above?
btw do you have issues applying the suggested changes with one click (e.g. github UI provides this)?